### PR TITLE
Fixed textarea overflow

### LINKED ISSUE #1980: Multi-line input textarea overflows container when words can't be broken
## Screenshots

image

## Version

Using the CDN.

## Describe the bug

Multi-line input textbox overflows parent container if words can't be broken when the styleoOption `sendBoxWrapText` is set to `true`,

### PR DESCRIPTION
**Pull request by tdurnford**
Fixes #1980 

## Changelog Entry
Fix [#1980](https://github.com/Microsoft/BotFramework-WebChat/issues/1980). Changed sendBoxTextArea styles to break words longer than the textarea, by [@tdurnford](https://github.com/tdurnford) in PR [#1986](https://github.com/Microsoft/BotFramework-WebChat/pull/1986)

## Description
Set the sendBoxTextArea's 'word-break' style option to 'break-word,' so that text wider than the textarea will wrap to the next line.

input-js-textarea-long-text-1-snap

- [x] Testing Added

### CODE REVIEW

File: packages/component/src/Styles/StyleSet/SendBoxTextArea.js [hunk 1]
```diff
@@ -26,9 +26,10 @@ export default function createSendBoxTextAreaStyle({
         fontFamily: 'inherit',
         fontSize: 'inherit',
         height: '100%',
-        overflowY: 'scroll',
+        overflowY: 'auto',
         whiteSpace: 'pre-wrap',
         width: 'inherit',
+        wordBreak: 'break-word'
```
Comment (compulim) [inline packages/component/src/Styles/StyleSet/SendBoxTextArea.js]: Should we following `styleOptions.messageActivityWordBreak`? @corinagum what do you think?

If yes, should we add more tests?

File: __tests__/input.js [hunk 2]
```diff
@@ -88,3 +88,15 @@ test('textarea send on enter', async () => {
 
   expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
 });
+
+test('textarea long text', async () => {
+  const { driver } = await setupWebDriver({ props: { styleOptions }});
+
```
Comment (compulim) [inline __tests__/input.js]: Remove empty line.

File: packages/component/src/Styles/StyleSet/SendBoxTextArea.js (hunk 1 above)
Comment (corinagum) [reply packages/component/src/Styles/StyleSet/SendBoxTextArea.js]: hmmm good point. However, if it's not 'break-word' it creates undesired behavior so I think we would want it to stay on 'break-word' always. 

Thoughts?

File: __tests__/input.js (hunk 2 above)
Comment (tdurnford) [reply __tests__/input.js]: I'm only seeing one empty line here.. Do you want me to remove line 94?

Comment (compulim) [reply __tests__/input.js]: No, it looks good. Bad eyes.

File: packages/component/src/Styles/StyleSet/SendBoxTextArea.js (hunk 1 above)
Comment (compulim) [reply packages/component/src/Styles/StyleSet/SendBoxTextArea.js]: My brain stuck last week. Please keep `break-word`.

**Pull request merged**